### PR TITLE
Support for multiple tooltips

### PR DESCRIPTION
**Pull request by mujx**
The tooltip objects are stored in arrays and I iterate over them to bind and unbind the event handlers.

I also added `top` as the default position to simplify error handling.

### CODE REVIEW

File: plugins/sigma.plugins.tooltips/sigma.plugins.tooltips.js [hunk 1]
```diff
@@ -118,15 +119,39 @@
...
     var self = this,
-        so = sigma.utils.extend(options.stage, settings.stage),
-        no = sigma.utils.extend(options.node, settings.node),
-        eo = sigma.utils.extend(options.edge, settings.edge),
         _tooltip,
         _timeoutHandle,
         _timeoutHideHandle,
+        _stageTooltips = [],
+        _nodeTooltips = [],
+        _edgeTooltips = [],
```
Comment (sheymann) [inline plugins/sigma.plugins.tooltips/sigma.plugins.tooltips.js]: Can you reset these 3 variables on kill to dereference the tooltip objects?

**Comment by mujx** [comment]
@sheymann Fixed. Do you want me to squash the commits?

**Comment by sheymann** [comment]
No it's fine as long as the merge is clean.

**Comment by sheymann** [comment]
thanks!

**Comment by sheymann** [comment]
note: this is a non-breaking change, old config will work.

**Pull request merged**